documentation/modules/ROOT/pages/1setup.adoc

```diff
@@ -67,8 +68,136 @@ oc login $(minishift ip):8443 -u admin -p admin
 
 NOTE: In this tutorial, you will often be polling the customer endpoint with `curl`, while simultaneously viewing logs via `stern` or `kubetail.sh` and issuing commands via `oc` and `istioctl`. Consider using three terminal windows.
 
-[#istioinstallation]
-== Istio installation script
+You have two options to install Istio in your environment. The option 1 is through the installation of https://docs.openshift.com/container-platform/3.10/servicemesh-install/servicemesh-install.html[Red Hat OpenShift Service Mesh], which includes Jaeger and Kiali in addition to Istio components. The option 2 is using the Istio source code from upstream community.
+
+[#redhatistioinstallation]
+== (Option 1) Red Hat OpenShift Service Mesh installation
+
+The Red Hat Istio uses an operator to install all Istio components and to set all permissions. You also get Jaeger and Kaili pre-installed in the bundle.
+
+Before start, you have to enable mutating and validating admission webhook plugins
+
+[source,bash]
+----
+#!/bin/bash
+
+minishift openshift config set --target=kube --patch '{
+    "admissionConfig": {
+        "pluginConfig": {
+            "ValidatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
+                    "kind": "WebhookAdmission",
+                    "kubeConfigFile": "/dev/null"
+                }
+            },
+            "MutatingAdmissionWebhook": {
+                "configuration": {
+                    "apiVersion": "apiserver.config.k8s.io/v1alpha1",
+                    "kind": "WebhookAdmission",
+                    "kubeConfigFile": "/dev/null"
+                }
+            }
+        }
+    }
+}'
+----
+
+You also need to set the system limits on mmap counts otherwise Elastic Search will fail. Jaeger uses Elastic Search and it is part of the Reed Hat Istio bundle. Elasticsearch uses a mmapfs directory by default to store its indices. The default operating system limits on mmap counts is likely to be too low, which may result in out of memory exceptions. To fix this you have to login into the minishift instance and do the following
```

Comment:
In big sentences like this add a break line after each period (.) char. In Asciidoc you need two line breaks to make a line break, so in that way is easy to review.

Reed -> Red

The official name is Elasticsearch, not Elastic Search.

I will add mmap, mmapfs between (`).

Reply:
Thanks for the review! All fixed.